Amazon.IonObjectMapper/Serializer/IonObjectSerializer.cs

```diff
@@ -48,8 +48,7 @@ public override object Deserialize(IIonReader reader)
                 if (ionConstructors.Count() > 1)
                 {
                     throw new InvalidOperationException(
-                        $"Only one constructor in class {this.targetType.Name} may be annotated " +
-                        "with the [IonConstructor] attribute and more than one was detected");
+                        $"Only one constructor in class {this.targetType.Name} may be annotated with the [IonConstructor] attribute and more than one was detected");
```

Comment:
Does this line exceed 120 characters?
Is StyleCop enforcing a max line length?

Reply:
StyleCop does not have a max line length rule

Reply:
More importantly, do we want to enforce a max line length?

Reply:
Maybe, but that's not for this PR.